front/components/ConnectorPermissionsTree.tsx

```diff
@@ -213,6 +219,8 @@ export function PermissionTree({
   canUpdatePermissions,
   onPermissionUpdate,
   showExpand,
+  displaySourceOverride,
```

Comment:
What about duplicating this to have a PermissionTree and a PermissionTreePoké, and `PermissionTreeChildren` would always receive the displaySource and connector permissions?

Reply:
Thanks for the suggestion, done